Contents/Sketch/lib/TextLayerSync.js

```diff
@@ -29,6 +34,19 @@ var TextLayerSync = function(layer)
 
     var layerName = _layer.name();
     var parts = layerName.split(':');
+
+    function wildcardMatch(key, store, language) {
+      var rex = new RegExp(key.replace('*', '.*'), 'i');
+      var candidateKeys = Object.keys(store);
+      var matchedKeys = candidateKeys.filter(function(key) {
+        return rex.test(key);
+      });
+      if (matchedKeys.length != 0) {
+        var randomMatchedKey = randomArrayItem(matchedKeys)
+        return store[randomMatchedKey][language];
+      }
```

Comment:
I can totally change this to be explicit with a `return null` at the end. 

> if this function always returns something

 A function with no return statement implicitly returns null. There isn't a difference between `function() {}` and `function() { return null }`. From the outside, they behave exactly the same . So adding `return null` would only be for explicit readability, which is reasonably salient.